include/mpd/recv.h

```diff
@@ -46,6 +46,14 @@ struct mpd_connection;
 extern "C" {
 #endif
 
+/**
+ * Reads the binary data from the server.  Returns NULL on error.
```

Comment:
When is it legal to call this function? The documentation does not say anything about it; can I call it after sending an arbitrary command?
What if this function call results in a short read? Does it block until all data is received? Does it block until at least one byte is received?

Reply:
updated documentation